### DISCUSSION TITLE
Filtering with multiple values

### DISCUSSION THREAD

**Discussion by RoYoMi** [CrowdStrike/psfalcon#276] [Q&A]
Using powershell module PSFalcon v2.2.4, powershell v7

I'm trying follow along with this documentation: https://github.com/CrowdStrike/psfalcon/wiki/Filtering-Results

The doc says the filter must be a string, and multiple values can be included as a quote-comma delimited list inside square brackets.

In this image we can see my individual hosts will return an ID, but when I try doing this as prescribed nothing gets returned.

![image](https://user-images.githubusercontent.com/8366040/215194100-854c257b-7e86-4aaa-9a5e-c69849c5ecc8.png)

Is there something I'm doing wrong here?

**Comment by bk-cs** [comment]
Yes, multiple filters are joined with commas (OR) or plus (AND):

```
# hostname1 or hostname2
Get-FalconHost -Filter "hostname:'hostname1',hostname:'hostname2'"
```
```
# hostname1 that is Windows
Get-FalconHost -Filter "hostname:'hostname1'+platform_name:'Windows'"
```

**Reply by bk-cs** [reply]
Also, hostnames are either partial match (quotes only) or exact match (including case, with `[]`).

```
# Find anything starting with host
Get-FalconHost -Filter "hostname:'host'" 
```
```
# Find only 'host'
Get-FalconHost -Filter "hostname:['host']" 
```

**Reply by bk-cs** [reply]
Try `Find-FalconHostname` if you want to find a list of hostnames.
